src/extension/openFileAtLocation.ts

```diff
@@ -41,6 +53,9 @@ getReactNativeWorkspaceForFile(filename).then(projectRootPath => {
 });
 
 function getReactNativeWorkspaceForFile(file: string): Q.Promise<string> {
+    if (workspace) {
```

Comment:
Should we consider passing in workspace as an optional parameter for this function?